src/main/java/teammates/ui/pagedata/StudentFeedbackResultsPageData.java

```diff
@@ -186,11 +186,6 @@ private FeedbackResultsResponseTable createResponseTable(FeedbackQuestionAttribu
                 // If the giver is the user, show the real name of the recipient
                 // since the giver would know which recipient he/she gave the response to
                 recipientName = bundle.getNameForEmail(response.recipient);
-            } else if (!isUserGiver
-                       && !bundle.isRecipientVisible(response)) {
-                // Hide anonymous recipient entirely to prevent student from guessing the identity
-                // based on responses from other response givers
-                recipientName = bundle.getAnonNameWithoutNumericalId(question.recipientType);
```

Comment:
Here

Reply:
Refer to the discussion in #8203, there is no need to hide the random identifier for recipient anymore.

Reply:
Check if the method `getAnonNameWithoutNumericalId` can be removed.